actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserProfileReadActor.java

```diff
@@ -312,6 +314,16 @@ private Response getUserProfileData(Request actorMessage) {
                   if (StringUtils.isNotBlank(s.get(JsonKey.ORIGINAL_EXTERNAL_ID))
                       && StringUtils.isNotBlank(s.get(JsonKey.ORIGINAL_ID_TYPE))
                       && StringUtils.isNotBlank(s.get(JsonKey.ORIGINAL_PROVIDER))) {
+                    if (JsonKey.DECLARED_DISTRICT.equals(s.get(JsonKey.ORIGINAL_ID_TYPE))
```

Comment:
declared-district, declared-state - are lowercase strings and I'd have preferred to use this Original column business only for encryption stuff.
We are moving towards original_columns only because they contain the actual value, in other DP reporting etc. So, this is okay - leaving a comment to raise my thought. No action required.